### ISSUE TITLE
Set up a demo instance of JIRA

### ISSUE THREAD

**Issue by twsswt** [twsswt/bug_buddy_jira_plugin#3]
Ask Tim for the money back once done (can be cloud hosted - check prices for this).

Add some example ticket data (try and make this sizable, so may take some time).

Add some dummy users (Say 4-5 with different roles).

Alternatively, look for example data to import (e.g. the dataset at http://openscience.us/repo/).

**Comment by stephen-brown-socs** [comment]
A demo instance of Jira now works on the lab machines. I ran into a disk quota error when trying to build the project using maven. It turns out maven downloads to a folder in the home directory, before copying over to the project location. In this instance I solved it by deleting some large, unused files. More info can be found at [1].

As of current time of writing, we can create a demo instance on any lab machine. However that involves multiple steps (download sdk, download git repo, build project). It would be nice to have a single script that does all three.

1. https://confluence.slac.stanford.edu/pages/viewpage.action?pageId=40350
